dyno-core/src/test/java/com/netflix/dyno/connectionpool/impl/ConnectionPoolImplTest.java

```diff
@@ -679,9 +679,14 @@ public String getName() {
 			
 
 			@Override
-			public String getKey() {
+			public String getStringKey() {
```

Comment:
can there not be two functions getKey() in Java, one returning string and one returning Byte[]

Reply:
I will complain about `Duplicate Method`.